docs/CONFIG-VARS.md

```diff
@@ -259,7 +259,7 @@ Each server element, like `foo = {}`, can contain none, some, or all of the para
 | backup_count | The number of automated backups to retain, from 1 to 365 | string | "7" | Take note this is a **COUNT** not number of days |
 | administrator_login | The Administrator Login for the PostgreSQL Server. Changing this forces a new resource to be created. | string | "pgadmin" | | |
 | administrator_password | The Password associated with the administrator_login for the PostgreSQL Server | string | "my$up3rS3cretPassw0rd" |  |
-| server_version | The version of the  PostgreSQL server instance | string | "13" | Supported values are 11-14 |
+| server_version | The version of the  PostgreSQL server instance | string | "13" | Refer to the [Viya 4 Deployment Guide](https://go.documentation.sas.com/doc/en/sasadmincdc/default/itopssr/p05lfgkwib3zxbn1t6nyihexp12n.htm?fromDefault=#p1wq8ouke3c6ixn1la636df9oa1u) to see what version of PostgreSQL your deployment will require |
```

Comment:
I would say it would list the supported versions not version required.

Reply:
Reworded it to say  "Refer to the Viya 4 Deployment Guide to see a list of supported PostgreSQL versions"